js/ui/drop_down_box.js

```diff
@@ -20,6 +22,29 @@ var DROP_DOWN_BOX_CLASS = "dx-dropdownbox",
  * @export default
  */
 var DropDownBox = DropDownEditor.inherit({
+    _supportedKeys: function() {
+        return extend({}, this.callBase(), {
+            tab: function(e) {
+                if(!this.option("opened")) {
+                    return;
+                }
+
+                var $tabbableElements = this._getTabbableElements(),
+                    $focusableElement = e.shiftKey ? $tabbableElements.last() : $tabbableElements.first();
+
+                $focusableElement && $focusableElement.focus();
+                e.preventDefault();
+            }
+        });
+    },
+
+    _getTabbableElements: function() {
+        return this._getElements().filter(selectors.tabbable);
```

Comment:
Change this._getElements().filter(selectors.tabbable) to this.content().find(selectors.tabbable) and remove unnecessary _getElements method.